lib/net.js

```diff
@@ -1131,6 +1203,19 @@ function lookupAndConnect(self, options) {
   }
   port |= 0;
 
+  if (typeof autoSelectFamily === 'boolean') {
+    autoSelectFamily = autoSelectFamily ? 250 : 0;
+  } else if (typeof autoSelectFamily !== 'undefined') {
+    autoSelectFamily = NumberParseInt(autoSelectFamily, 10);
+    validateInt32(autoSelectFamily, 0);
```

Comment:
Do we need the `parseInt`? Shouldn't we go straight to `validateInt32`?

Reply:
Actually yes. I was trying to be more lenient but I think it will just open the door to horrible stuff. Removing it.